config/routes.rb

```diff
@@ -28,8 +28,8 @@
     get  'progress'
   end
 
-  resource :tag_search, only: [:new] do
-    get 'results' => 'tag_searches#results', as: "results_of"
+  resource :bulk_tag, only: [:new] do
+    get 'results' => 'bulk_tags#results', as: "results_of"
```

Comment:
The helper method 'results_of_bulk_tag_path' is a little misleading. Reckon it'd be better to change the `as: ` value to `search_results_for`.